source/guide_goaccess.rst

```diff
@@ -0,0 +1,199 @@
+.. highlight:: console
+
+.. author:: FM <git.fm@mmw9.de>
+
+########
+GoAccess
+########
+
+GoAccess_ is an open source realtime web log analyzer and viewer. It use the standard web server log file as source to build a readable graphical HTML output as web page (dashboard) or an overview in the shell. 
+
+This documentation describe the way to let create a static web page as overview, which will be updated every hour with a cron job. More details are on the `man page`_ and on GitHub_ available.
+
+License
+=======
+
+GoAccess is released under the `MIT License`_
+
+Installation
+============
+
+Step 1 - Download and Extract the Source Code
+---------------------------------------------
+
+Create a working directory.
+
+::
+
+ [isabell@stardust ~]$ mkdir ~/filebase/
+ [isabell@stardust ~]$ cd ~/filebase/
+ [isabell@stardust ~]$
+
+Download the actual source code from GitHub. 
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/allinurl/goaccess.git
+ [isabell@stardust ~]$
+
+Step 2 - Source Code Configuration, Compiling and Installation
+--------------------------------------------------------------
+
+Prepare the compile environment.
+
+::
+
+ [isabell@stardust ~]$ cd ~/filebase/goaccess/
+ [isabell@stardust ~]$ autoreconf -fiv
+ [isabell@stardust ~]$
+
+Configure and compile the source code with the commands ``configure``, ``make`` and finally install it with ``make install``.
+
+.. note:: Please use single steps instead of combining all three in one process to see and identify possible errors.
+
+Before we start, we have to consider some aspects, especially a shared hosting environment like Uberspace:
+
+ * ``--prefix=/home/isabell``: Installation target for your personal Uberspace.
```

Comment:
Please use `--prefix=$HOME` here and in the actual command to spare users the manual substitution.